.github/workflows/ci-minikbe.yml

```diff
@@ -0,0 +1,71 @@
+name: ci-minikube
+
+on:
+  push:
+    branches:
+      - main
+      - dev
+    pull_request: 
+      branches:
+        - main
+        - dev
+
+jobs:
+  build:
+    name: Auto-testing Framework / ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest,ubuntu-18.04]
+    steps:
+      - name: Check Docker Version
+        run: uname -r
+      
+      - uses: actions/checkout@v2
+
+      - name: Install Minikube
+        run: |
+          set -x
+          curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube_latest_amd64.deb
+          sudo dpkg -i minikube_latest_amd64.deb
+          rm minikube_latest_amd64.deb
+          curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+          sudo mv kubectl /usr/bin
+          sudo chmod 755 /usr/bin/kubectl
+      
+      - name: Start Minikube
+        run: minikube start
+      
+      - name: Apply VM and Hostpolicy CRD
+        run: |
+          sleep 30
```

Comment:
why sleep 30?

Reply:
it need some time to start correctly , or it will fail